moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -333,40 +332,32 @@ const ProjectFundingTable = () => {
   );
 
   /**
-   * Component for dropdown select for Funds
+   * Autocomplete component for Funds
    * @param {*} props
    * @returns {React component}
    */
-  const FundSelectComponent = (props) => (
-    <Select
-      id="moped_funds"
-      value={
-        props.value?.fund_id
-          ? `${props.value.fund_id} | ${props.value.fund_name}`
-          : ""
-      }
+  const FundAutocompleteComponent = (props) => (
```

Comment:
I can see where John is going, and I think the autocompletes could be a bit more abstracted by passing some functions as props that know how to get the options, selected option, and values for each type of data. That way the component doesn't have to be concerned about specific key names for options or values that it uses.

I ran into this the other day too and realized that our DB column names aren't always as predictable as they could be when it comes to record `id` and `name` which makes features like this tricky.